openwisp_monitoring/db/backends/influxdb/queries.py

```diff
@@ -1,46 +1,62 @@
 chart_query = {
-    'uptime': (
-        "SELECT MEAN({field_name})*100 AS uptime FROM {key} WHERE "
-        "time >= '{time}' AND content_type = '{content_type}' AND "
-        "object_id = '{object_id}' GROUP BY time(1d)"
-    ),
-    'packet_loss': (
-        "SELECT MEAN(loss) AS packet_loss FROM {key} WHERE "
-        "time >= '{time}' AND content_type = '{content_type}' AND "
-        "object_id = '{object_id}' GROUP BY time(1d)"
-    ),
-    'rtt': (
-        "SELECT MEAN(rtt_avg) AS RTT_average, MEAN(rtt_max) AS "
-        "RTT_max, MEAN(rtt_min) AS RTT_min FROM {key} WHERE "
-        "time >= '{time}' AND content_type = '{content_type}' AND "
-        "object_id = '{object_id}' GROUP BY time(1d)"
-    ),
-    'wifi_clients': (
-        "SELECT COUNT(DISTINCT({field_name})) AS wifi_clients FROM {key} "
-        "WHERE time >= '{time}' AND content_type = '{content_type}' "
-        "AND object_id = '{object_id}' GROUP BY time(1d)"
-    ),
-    'traffic': (
-        "SELECT SUM(tx_bytes) / 1000000000 AS upload, "
-        "SUM(rx_bytes) / 1000000000 AS download FROM {key} "
-        "WHERE time >= '{time}' AND content_type = '{content_type}' "
-        "AND object_id = '{object_id}' GROUP BY time(1d)"
-    ),
-    'memory': (
-        "SELECT 100 * MEAN(percent_used) AS memory_usage "
-        "FROM {key} WHERE time >= '{time}' AND content_type = '{content_type}' "
-        "AND object_id = '{object_id}' GROUP BY time(1d)"
-    ),
-    'cpu': (
-        "SELECT 100 * MEAN(cpu_usage) AS CPU_load FROM {key} WHERE "
-        "time >= '{time}' AND content_type = '{content_type}' AND "
-        "object_id = '{object_id}' GROUP BY time(1d)"
-    ),
-    'disk': (
-        "SELECT 100 * MEAN(used_disk) AS disk_usage FROM {key} WHERE "
-        "time >= '{time}' AND content_type = '{content_type}' AND "
-        "object_id = '{object_id}' GROUP BY time(1d)"
-    ),
```

Comment:
I think this was fine, it could have been manipulated with a for loop to accommodated with a `for` loop.
But I'm merging anyway because I want to move on, if you want to do this improvement, don't hesitate.